cmake/Reprojucer.cmake

```diff
@@ -1220,7 +1220,7 @@ function(jucer_project_end)
   endif()
 
   if(WIN32 AND NOT JUCER_PROJECT_TYPE STREQUAL "Static Library")
-    if(NOT(DEFINED JUCER_VERSION AND JUCER_VERSION VERSION_LESS 5.2))
+    if(DEFINED JUCER_COMPANY_COPYRIGHT)
```

Comment:
Who is defining `JUCER_COMPANY_COPYRIGHT`?

Reply:
It's done by `jucer_project_settings()`  https://github.com/McMartin/FRUT/blob/1487c70dd613a05d1aa0cb3cbca130c5b08d8c42/cmake/Reprojucer.cmake#L199

when `tag` equals `COMPANY_COPYRIGHT`.